aws/infrastructure/index.ts

```diff
@@ -33,29 +33,31 @@ if (config.vpcId === undefined) {
 
 // Import the identity stack for its roles.
 const identityStack = new pulumi.StackReference("identityStack", { name: config.identityStack });
+const kubeAppRole = identityStack.getOutput("kubeAppRoleArn");
+kubeAppRole.apply(role => {
+    if (role === undefined) {
+        throw Error(`Stack output ${identityStack}.kubeAppRoleArn is undefined`);
+    }
+});
 
 // Create the EKS cluster itself.
 const eksCluster = new eks.Cluster("eksCluster", {
     vpcId: vpcId,
-    subnetIds: [ ...publicSubnetIds, ...(privateSubnetIds || []) ],
+    subnetIds: [...publicSubnetIds, ...(privateSubnetIds || [])],
     roleMappings: [
+        // TODO: Find a slightly more restrictive permission for this.
         {
```

Comment:
For my own curiosity, what alternative, restrictive perm are you thinking? To create the cluster, you inenvitably need to be in the `system:masters` group [to perform all cluster ops](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)

Reply:
I'm not sure what we want to do here yet because I am not sure how to map AWS IAM -> k8s identity. I think it is not unlikely that we'll need this to be `system:masters`, but it also might not be the case.